foundations/installations/command_line_basics.md

```diff
@@ -45,11 +45,11 @@ Many of these resources assume you're using a Mac or Linux environment. If you d
 
 1. Visit [The Unix Shell](https://swcarpentry.github.io/shell-novice/) course designed by the Software Carpentry Foundation. There you will find a full complement of lessons on using the CLI, but for now just focus on completing the following lessons:
     
-    * Setup (Follow instructions in `Download files` section only, this is all you need as you have the required software already)
-    * Introducing the Shell
-    * Navigating Files and Directories
-    * Working With Files and Directories
-    * Pipes and Filters
+    - Setup (Follow instructions in `Download files` section only, this is all you need as you have the required software already).
+    - Introducing the Shell.
+    - Navigating Files and Directories.
+    - Working With Files and Directories.
+    - Pipes and Filters.
```

Comment:
These items should have a single whitespace removed from each. Additionally I don't believe we need the punctuation at the end of these, but if others feel differently then we can keep them 

```suggestion
   - Setup (`Download files` section only)
   - Introducing the Shell
   - Navigating Files and Directories
   - Working With Files and Directories
   - Pipes and Filters
```

Updated the verbiage for the Setup item as well just to make it more succinct

Reply:
- Setup (`Download files` section only)

I've seen  some student on the server asked whether they need to do the  [Install software](https://swcarpentry.github.io/shell-novice/#install-software) section also. So above suggestion might confuse students even more.

I would like to know keep line 48 as it is

- Setup (Follow instructions in `Download files` section only, this is all you need as you have the required software already)

Or modify it by adding 'installed' to the end of the sentence 

- Setup (Follow instructions in `Download files` section only, this is all you need as you have the required software already installed)

Or update line 48 as suggested

 - Setup (`Download files` section only)

And i agree with removing single white space and removing end punctuation suggestions.

Reply:
I'd be fine with either approach knowing that context. We could also just update the first bullet to "Download files" instead of "Setup", and directly link to each section in this list as well.